stable/elasticsearch/Chart.yaml

```diff
@@ -1,6 +1,6 @@
 name: elasticsearch
 home: https://www.elastic.co/products/elasticsearch
-version: 1.22.0
```

Comment:
Minor version bump as it is a new feature?

Reply:
sure